src/main/resources/staff/templates/document/sealedClaim.html

```diff
@@ -277,6 +277,22 @@
       </tr>
       {% endfor %}
       {% endif %}
+
+      {% if claim.evidences is defined %}
```

Comment:
I believe that we should always display "Evidence" but sometimes with empty list.

It's for robotics team to make their life easier. Unless this approach was changed and I didn't know about it :)

Reply:
This is for the existing claims when we didn't have evidence option.